conf/agent/agent.conf

```diff
@@ -13,8 +13,9 @@ plugins {
         plugin_data {
         }
     }
-    KeyManager "memory" {
+    KeyManager "disk" {
```

Comment:
I personally welcome this change for dev work. @evan2645 is there a reason we would want to keep this memory?

Reply:
I was looking at this yesterday... I am not sure. Agent will try to use its old SVID which may no longer be valid... Let's just give it a shot and see, maybe we can improve some behavior there if we run into any pain.